core/src/main/java/com/novoda/noplayer/exoplayer/ExoPlayerTwoImpl.java

```diff
@@ -37,12 +44,14 @@
                      PlayerListenersHolder listenersHolder,
                      ExoPlayerForwarder exoPlayerForwarder,
                      LoadTimeout loadTimeoutParam,
-                     Heart heart) {
+                     Heart heart,
+                     ExoPlayerTrackSelector exoPlayerTrackSelector) {
         this.exoPlayer = exoPlayer;
         this.listenersHolder = listenersHolder;
         this.loadTimeout = loadTimeoutParam;
         this.forwarder = exoPlayerForwarder;
         this.heart = heart;
+        this.exoPlayerTrackSelector = exoPlayerTrackSelector;
```

Comment:
Why do we have to know about the exoPlayerTrackSelector here ? Shouldn't we delegate to exoPlayerFacade who already deals with this ?